frontend/packages/enduser-frontend/src/views/applications/daycare-application/daycare-application.vue

```diff
@@ -166,18 +166,12 @@ SPDX-License-Identifier: LGPL-2.1-or-later
                         "
                       ></div>
                     </c-message-box>
-                    <div v-if="attachmentsEnabled">
```

Comment:
Sidenote regarding `<c-checkbox v-model="model" name="urgent">`: this should use a `:label` instead of text element, like in:

```vue
<c-checkbox
                      v-model="model"
                      name="extendedCare"
                      :label="
                        $t(`form.${type}-application.service.extended.label`)
                      "
                    >
```

as currently clicking the text "Hakemus on kiireellinen" doesn't actually toggle the checkbox (even though it should and even shows the pointer cursor).